tests/phpunit/api/v3/SettingTest.php

```diff
@@ -111,11 +111,9 @@ public function testGetFieldsFilters($version) {
 
   /**
    * Test that getfields will filter on group.
-   * @param int $version
-   * @dataProvider versionThreeAndFour
    */
-  public function testGetFieldsGroupFilters($version) {
-    $this->_apiversion = $version;
+  public function testGetFieldsGroupFilters() {
+    $this->_apiversion = 3;
```

Comment:
`group` is a deprecated property so APIv4 doesn't need to test it.

Reply:
Does apiv4 still permit filtering on it if set. We kinda deprecated it as a required key but kept the idea you could filter by it (e.g if you wanted to present all the settings defined by one extension on a page). I can't recall if it was reliable or not

Reply:
No. I was following [the docs ](https://docs.civicrm.org/dev/en/latest/framework/setting/#deprecated-properties)which state it is a deprecated property. I didn't want APIv4 returning stuff that's already deprecated. That means it wouldn't be able to filter on it either.